.github/workflows/rti-docker.yml

```diff
@@ -26,3 +26,10 @@ jobs:
           tag: ${{ env.lf_version }}
           DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
           DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Update latest (released versions only)
+        uses: ./.github/actions/push-rti-docker
+        with:
+          tag: latest
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+        if: ${{ env.lf_version != *-snapshot }}
```

Comment:
```suggestion
        if: ${{ ! endsWith( env.lf_version, '-snapshot' ) }}
```

Reply:
Looks good to me, but how do we confirm that it works?

Reply:
I think [this](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/expressions#endswith) might be what you are looking for.

Reply:
I think the only way is to do a fork and merge to the master branch in the fork

Reply:
Right. Could you perhaps carry out this experiment?

Reply:
We could also just publish another release and see if it works. If it doesn't, we could update `latest` manually.

Reply:
It seems to be working in my fork in [this](https://github.com/Depetrol/lingua-franca/actions/runs/10222853768/job/28288178683) run for [this](https://github.com/Depetrol/lingua-franca/pull/22) PR. An image with `latest` tag has been pushed to [my dockerhub](https://hub.docker.com/r/depetrol/rti/tags).

Reply:
Perfect!